### ISSUE TITLE
opendistro_security-1.1.0.0.zip not found

### ISSUE THREAD

**Issue by shabtaisharon** [opendistro-for-elasticsearch/security#71]
I'm trying to install the plugin to my ES v7.1.1 and i'm getting the below error:

```
➜  elasticsearch-7.1.1 git:(ESCP-252) ✗ ./bin/elasticsearch-plugin install -b com.amazon.opendistroforelasticsearch:opendistro_security:1.1.0.0
-> Downloading com.amazon.opendistroforelasticsearch:opendistro_security:1.1.0.0 from maven central
Exception in thread "main" java.io.FileNotFoundException: https://repo1.maven.org/maven2/com/amazon/opendistroforelasticsearch/opendistro_security/1.1.0.0/opendistro_security-1.1.0.0.zip
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1890)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1492)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:263)
	at org.elasticsearch.plugins.InstallPluginCommand.downloadZip(InstallPluginCommand.java:379)
	at org.elasticsearch.plugins.InstallPluginCommand.downloadAndValidate(InstallPluginCommand.java:454)
	at org.elasticsearch.plugins.InstallPluginCommand.download(InstallPluginCommand.java:264)
	at org.elasticsearch.plugins.InstallPluginCommand.execute(InstallPluginCommand.java:229)
	at org.elasticsearch.plugins.InstallPluginCommand.execute(InstallPluginCommand.java:216)
	at org.elasticsearch.cli.EnvironmentAwareCommand.execute(EnvironmentAwareCommand.java:86)
	at org.elasticsearch.cli.Command.mainWithoutErrorHandling(Command.java:124)
	at org.elasticsearch.cli.MultiCommand.execute(MultiCommand.java:77)
	at org.elasticsearch.cli.Command.mainWithoutErrorHandling(Command.java:124)
	at org.elasticsearch.cli.Command.main(Command.java:90)
	at org.elasticsearch.plugins.PluginCli.main(PluginCli.java:47)
```

**Comment by shabtaisharon** [comment]
I can see that you don't have v1.1.0.0 uploaded yet to maven [https://mvnrepository.com/artifact/com.amazon.opendistroforelasticsearch/opendistro_security](https://mvnrepository.com/artifact/com.amazon.opendistroforelasticsearch/opendistro_security)

![Screen Shot 2019-07-03 at 11 20 54 AM](https://user-images.githubusercontent.com/14626506/60611918-ba65fa00-9d84-11e9-9457-acb7409fabbd.png)

Can someone please confirm if v1.0.0.0 will work with ES v7.1.1?

**Comment by shabtaisharon** [comment]
followed the instructions on how to build the plugin [here](https://github.com/opendistro-for-elasticsearch/security-parent/blob/master/README.md#build)

and now I get the following error message:
```
➜  elasticsearch-7.1.1 git:(ESCP-252) ✗ ./bin/elasticsearch-plugin install /Users/sharons/github/opendistro/security/target/releases/opendistro_security-1.1.0.0.zip
A tool for managing installed elasticsearch plugins

Commands
--------
list - Lists installed elasticsearch plugins
install - Install a plugin
remove - removes a plugin from Elasticsearch

Non-option arguments:
command

Option         Description
------         -----------
-h, --help     show help
-s, --silent   show minimal output
-v, --verbose  show verbose output
ERROR: Unknown plugin /Users/sharons/github/opendistro/security/target/releases/opendistro_security-1.1.0.0.zip
```

you can see that i have the zip file here:
```
➜  releases git:(master) pwd
/Users/sharons/github/opendistro/security/target/releases
➜  releases git:(master) ll
total 246488
drwx------  88 sharons  SLDOMAIN\Domain Users   2.8K Jul  3 12:46 opendistro_security-1.1.0.0
-rw-r--r--   1 sharons  SLDOMAIN\Domain Users    47M Jul  3 12:40 opendistro_security-1.1.0.0-securityadmin-standalone.tar.gz
-rw-r--r--   1 sharons  SLDOMAIN\Domain Users    47M Jul  3 12:40 opendistro_security-1.1.0.0-securityadmin-standalone.zip
-rw-r--r--   1 sharons  SLDOMAIN\Domain Users   186B Jul  3 12:40 opendistro_security-1.1.0.0-securityadmin-standalone.zip.sha512
-rw-r--r--@  1 sharons  SLDOMAIN\Domain Users    26M Jul  3 12:40 opendistro_security-1.1.0.0.zip
-rw-r--r--   1 sharons  SLDOMAIN\Domain Users   161B Jul  3 12:40 opendistro_security-1.1.0.0.zip.sha512
```

**Comment by aetter** [comment]
Hi @shabtaisharon, correct, we haven't released 1.1.0 yet. If you're in a hurry and don't mind upgrading again when we release 1.1.0, try `sudo bin/elasticsearch-plugin install https://d3g5vo6xdbdb9a.cloudfront.net/downloads/elasticsearch-plugins/opendistro-security/opendistro_security-1.1.0.0.zip`.

**Comment by shabtaisharon** [comment]
thank you @aetter
